doc/Language/statement-prefixes.pod6

```diff
@@ -65,12 +65,12 @@ This code is around 3x faster than the bare for. But there are a couple of
 caveats here:
 
 =item The operation inside the loop should take enough time to make threading
-make sense.
+makes sense.
```

Comment:
Not really. "make sense" makes more sense here.

Reply:
+1, but I would say that the overall sentence structure is hmm. "to make threading make sense"... Maybe we can rewrite it somehow?

Reply:
... "for threading to make sense"